### DISCUSSION TITLE
Template Standalone

### DISCUSSION THREAD

**Discussion by hassankhalil99** [primefaces/community#1886] [PrimeNG Templates] [answered]
hello dears , I'm interested  to buy the Poseidon template for my angular project . I'm just wondering if it's currently based on angular module,
if it's a module template , I will be updated when you change it to standalone to benefit from it ? or should I buy it again 
thank you in advance

**Comment by mehmetcetin01140** [comment, marked as answer]
Hi, it's currently used as a module in the current version. Purchasing the template once is sufficient. You can benefit from all updates. We plan to convert the templates to standalone in the v18 update.

**Comment by anonymousdplantgit** [comment]
Hi there, any idea when the 18v is planned to be released ?
